src/react/components/pages/editorPage/editorTagsInput.tsx

```diff
@@ -36,38 +23,20 @@ export default class EditorTagsInput extends TagsInput<IEditorTagsInputProps> {
         );
     }
 
-    /**
-     * Calls the onTagClick handler if not null with clicked tag
-     * @param event Click event
-     */
-    protected handleTagClick(event) {
-        const text = this.getTagIdFromClick(event);
-        const tag = this.getTag(text);
-        if (event.ctrlKey) {
-            this.openEditModal(tag);
-        } else if (event.shiftKey && this.props.onTagShiftClick) {
-            // Calls provided onTagShiftClick
-            this.props.onTagShiftClick(this.toItag(tag));
-        } else if (this.props.onTagClick) {
-            // Calls provided onTagClick function
-            this.props.onTagClick(this.toItag(tag));
-        }
-    }
-
     private indexOfTag(id: string): number {
         let index = -1;
-        if (this.state) {
-            index = this.state.tags.findIndex((tag) => tag.id === id);
-            if (index < 0) {
-                index = this.state.tags.length + 1;
+        if (this.state) { // if stat exists
+            index = this.state.tags.findIndex((tag) => tag.id === id); // return index based on tag.id or -1
+            if (index < 0) { // if not found, i.e. -1
+                index = this.state.tags.length + 1; // create new final array index and assign to index, i.e. 2 or 3
             }
-        } else {
-            index = this.props.tags.findIndex((tag) => tag.name === id);
+        } else { // if there is no state
+            index = this.props.tags.findIndex((tag) => tag.name === id); // check tags in props and return index or -1
         }
-        if (index < 0) {
+        if (index < 0) { // if neither of the above were triggered, throw error
             throw new Error(`No tag by id: ${id}`);
         }
-        index += 1;
-        return (index === 10) ? 0 : index;
+        index += 1; // whatever the index, increment by one to avoid zero-base in display
+        return (index === 10) ? 0 : index; // return index or 0, because max hotkeys are 10?
```

Comment:
nit: remove `?`. If you were still wondering about this, yes, the max number of hot keys are 10, and we want `0` to be the last key that shows. We could probably make that a little more clear and avoid having a magic number in the function, but I think it's fine for now

Reply:
I meant to remove all of these comments. they were just to help me find where an error was coming from. BUT I was think that maybe this should be incorporated into the TagsInput component in vott-react and this component removed from here entirely. The only difference is displaying the index numbers, which seems fine to have in the project settings as well.

Reply:
If one of the hotkeys is going to be 0, we could just leave the zero-indexing